bower.json

```diff
@@ -3,9 +3,9 @@
   "version": "0.0.1",
   "dependencies": {
     "jquery": "1.11.2",
-    "digitalmarketplace-frontend-toolkit": "git://github.com/alphagov/digitalmarketplace-frontend-toolkit#v10.0.0",
+    "digitalmarketplace-frontend-toolkit": "git://github.com/alphagov/digitalmarketplace-frontend-toolkit#v11.2.0",
     "govuk_template": "https://github.com/alphagov/govuk_template/releases/download/v0.14.1/jinja_govuk_template-0.14.1.tgz",
-    "digital-marketplace-ssp-content": "git://github.com/alphagov/digital-marketplace-ssp-content#15fcaeab5b24893a04bece14a56ff2fa425e5633",
+    "digital-marketplace-ssp-content": "git://github.com/alphagov/digital-marketplace-ssp-content#d47defa52ee51654a42b46d7d6e2bca8c8c14f50",
```

Comment:
This can be https://github.com/alphagov/digitalmarketplace-frameworks/releases/tag/v0.0.1 now